fixtures/location/all_locations_fixtures.py

```diff
@@ -0,0 +1,81 @@
+
+all_locations_query = '''
+                                    {
+                            allLocations{
+                                edges{
```

Comment:
Update this to reflect removed edges and nodes

Reply:
Thanks this has been resolved.